kani-driver/src/main.rs

```diff
@@ -77,6 +82,11 @@ fn cargokani_main(input_args: Vec<OsString>) -> Result<()> {
         return assess::run_assess(session, assess::AssessArgs::default());
     }
 
+    if !session.args.common_args.quiet {
+        let kani_version = print_kani_version();
```

Comment:
Shouldn't this method print the version? It has `print` in the name. :)

Reply:
True, it prints now.